Correct typo in Semantic/ValueTupleTests. #12010

bnordli opened the pull request:
Fixes #12010 

Test output after change:

```
------ Test started: Assembly: Roslyn.Compilers.CSharp.Semantic.UnitTests.dll ------

2 passed, 0 failed, 0 skipped, took 1,25 seconds (xUnit.net 2.1.0 build 3179).

```

gafter commented:
@VSadov FYI

bnordli commented:
The branch of the PR has been rebased onto master. I assume that is encouraged on such small PRs.

dnfclas commented:
__@bnordli__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, DNFBOT;

bnordli commented:
No interest in this? Should I just close it?

gafter commented:
Why isn't this test failing for us?

bnordli commented:
Isn't it? I ran all unit tests in the Compiler solution manually using TestDriven.Net, and this one and #12094 were the only tests that failed for me.

Lots of tests from `Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics` are missing from the CI projects, at least according to this log, which is the last run ATM:
http://dotnet-ci.cloudapp.net/job/dotnet_roslyn/job/master/job/windows_debug_unit32_prtest/lastBuild/testReport/Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics/ (It lists four classes, while the namespace contains 12 files.)

I will happily add more tests to the suites, if that is the correct fix, and if I get a pointer to which file contains the CI definitions.

gafter commented:
@jcouv Hey Julien, do you know why these ValueTuple tests, which have bugs, are not failing in our daily builds?

jcouv commented:
Let me check. The test does seem to have a bug and should fail every time.

jcouv commented:
@bnordli Thanks for finding this issue. 
Upon investigation, it seems the problem is that the `ValueTupleTests` class is not marked as public. So it doesn't get run by the test automation, although it can be run individually as you did.
Would you mind adding that fix to your PR?

` public class ValueTupleTests : CompilingTestBase`

bnordli commented:
No problem!

I added the `public` modifier to the test class, and rebased the branch onto the recent master.

The test ran successfully, according to this report: http://dotnet-ci.cloudapp.net/job/dotnet_roslyn/job/master/job/windows_debug_unit32_prtest/1309/testReport/Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics/ValueTupleTests/

jcouv commented:
LGTM. I'll let @gafter provide a second review.
Thanks again for catching this!

VSadov commented:
LGTM. Thanks!!! Good catch.
